seqr/views/apis/saved_variant_api.py

```diff
@@ -332,3 +333,22 @@ def update_variant_main_transcript(request, variant_guid, transcript_id):
     update_model_from_json(saved_variant, {'selected_main_transcript_id': transcript_id}, request.user)
 
     return create_json_response({'savedVariantsByGuid': {variant_guid: {'selectedMainTranscriptId': transcript_id}}})
+
+
+@login_and_policies_required
+def get_clingen_allele_id(request, hgvsc):
```

Comment:
I don't know which file the new backend API should go to. I put it in the `saved_variant_api` for now.